README.md

```diff
@@ -93,10 +71,44 @@ getPackageName ('package.json')
 
 ### Global Bundle (CDN)
 
-Fluture is hosted in full with all of its dependencies at
-https://cdn.jsdelivr.net/gh/fluture-js/Fluture@12.2.1/dist/bundle.js
+To load Fluture directly into a browser, a code pen, or [Deno][], use one of
+the following downloads from JSDelivr. They are single files that come with all
+of Fluture's dependencies pre-bundled.
+
+- [Fluture Script][]: A JavaScript file that adds `Fluture` to the global
+  scope. Ideal for older browsers and code pens.
+- [Fluture Script Minified][]: The same as above, but minified.
+- [Fluture Module][]: An EcmaScript module with named exports. Ideal for Deno
+  or modern browsers.
+- [Fluture Module Minified][]: A minified EcmaScript module without TypeScript typings. Not recommended for Deno.
 
-This script will add `Fluture` to the global scope.
+[Fluture Script]: https://cdn.jsdelivr.net/gh/fluture-js/Fluture@12.2.1/dist/bundle.js
+[Fluture Script Minified]: https://cdn.jsdelivr.net/gh/fluture-js/Fluture@12.2.1/dist/bundle.min.js
+[Fluture Module]: https://cdn.jsdelivr.net/gh/fluture-js/Fluture@12.2.1/dist/module.js
+[Fluture Module Minified]: https://cdn.jsdelivr.net/gh/fluture-js/Fluture@12.2.1/dist/module.min.js
+
+### CommonJS Module
+
+Although the Fluture source uses the EcmaScript module system,
+the `main` file points to a CommonJS version of Fluture.
+
+On older environments you may need to polyfill one or more of the following
+functions: [`Object.create`][JS:Object.create],
+[`Object.assign`][JS:Object.assign] and [`Array.isArray`][JS:Array.isArray].
+
+```js
+const fs = require ('fs')
+const Future = require ('fluture')
+
+const getPackageName = function (file) {
+  return Future.node (function (done) { fs.readFile (file, 'utf8', done) })
+  .pipe (Future.chain (Future.encase (JSON.parse)))
+  .pipe (Future.map (function (x) { return x.name }))
+}
```

Comment:
It's a little strange to see `const` alongside function expressions. Is there a reason to avoid arrow functions here?

Reply:
Hm. You're right. This example was meant to illustrate what Fluture usage looks like in older runtimes. I guess I missed the `const`.